src/pages/operators/index.tsx

```diff
@@ -423,6 +428,7 @@ const Operators: React.VFC<Props> = (props) => {
             </ClassSubclassMenuItem>
           ))}
       </Menu>
+      {(selectedProfession !== null || selectedSubProfessionId !== null) && <button className="reset-filters-button" onClick={handleResetFilter}>Reset</button>}
```

Comment:
Let's just loosen this to `!= null` instead of `!==` (so as to match `undefined` too). I don't actually expect the two vars to be undefined, but we don't really lose anything by doing so.

Reply:
Got it!
Updated